executable-spec/src/Cardano/Ledger/Spec/STS/Update/Ideation.hs

```diff
@@ -191,18 +193,25 @@ instance
 
   sigGen
     _traceGenEnv
-    Env{ k, currentSlot, participants }
+    Env{ k, currentSlot, asips, participants }
     St{ wssips, wrsips, subsips } =
       case Set.toList $ range $ stableCommits ◁ subsips of
```

Comment:
Couldn't we have a situation in which there aren't stable commits but there are active SIPs? In such case we could generate a submission as well.

Reply:
I assume you want to say "In such case we could generate a vote as well." Right?
And I think that you are right.  I have to change the code to generate also votes for this case, only if there are active SIP whose voting end has not been reached yet.

Done.OK